src/group_theory/quotient_group.lean

```diff
@@ -80,6 +80,47 @@ instance : group (quotient N) :=
 @[to_additive quotient_add_group.mk' "The additive group homomorphism from `G` to `G/N`."]
 def mk' : G →* quotient N := monoid_hom.mk' (quotient_group.mk) (λ _ _, rfl)
 
+@[to_additive, simp]
+lemma coe_mk' : (mk' N : G → quotient N) = coe := rfl
+
+@[to_additive, simp]
+lemma mk'_apply (x : G) : mk' N x = x := rfl
+
+@[to_additive]
+lemma mk'_eq_mk'_iff {x y : G} :  mk' N x = mk' N y ↔  x⁻¹*y ∈ N :=
+quotient_group.eq
+
+@[to_additive]
+lemma mk'_eq_mk'_iff' {x y : G} :  mk' N x = mk' N y ↔  y*x⁻¹ ∈ N :=
+begin
+  rw mk'_eq_mk'_iff,
+  split,
+  { intro h,
+    simpa using nN.conj_mem _ h x },
+  { intro h,
+    convert nN.conj_mem _ h x⁻¹ using 1,
+    rw [mul_assoc, inv_inv, mul_assoc y, mul_left_inv, mul_one] }
+end
+
+omit nN
+
+lemma _root_.quotient_add_group.mk'_eq_mk'_iff_sub {G : Type*} [add_group G]
+  [H : add_subgroup G] [H.normal] {x y : G} :
+quotient_add_group.mk' H x = quotient_add_group.mk' H y ↔ y - x ∈ H :=
+by rw [quotient_add_group.mk'_eq_mk'_iff', sub_eq_add_neg]
```

Comment:
This lemma can be proved with `quotient_add_group.eq_iff_sub_mem` which appears slightly further down the file.